content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+
+[Session Persistence](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SessionPersistence) is being introduced to Gateway API via a new policy
+([BackendLBPolicy](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.BackendLBPolicy))
+for Service-level configuration and as fields within HTTPRoute
+and GRPCRoute for route-level configuration. The BackendLBPolicy and route-level
+APIs provide the same session persistence configuration, including session
+timeouts, session name, session type, and cookie lifetime type.
+
+Leading Contributors: Grant Spence, Gina Yeh
+
+#### [TLS Terminology Clarifications](https://gateway-api.sigs.k8s.io/geps/gep-2907/)
+
+As part of a broader goal of making our TLS terminology more consistent
+throughout the API, we've introduced some breaking changes to BackendTLSPolicy.
+This has resulted in a new API version (v1alpha3) and will require any existing
+users of this policy to uninstall the v1alpha2 version before installing this
+newer version.
+
+Any references to v1alpha2 BackendTLSPolicy fields will need to be updated to
+v1alpha3. Specific changes to fields include:
+- TargetRef is changed to allow multiples: BackendTLSPolicySpec.TargetRef
+becomes a slice named `TargetRefs`.
+- TLS is changed to `Validation`, and the type name changes from
+BackendTLSPolicyConfig to  `BackendTLSPolicyValidation`.
+- CACertRefs becomes `CACertificateRefs` in what is now `BackendTLSPolicyValidation`
+- WellKnownCACerts becomes `WellKnownCACertificates` and the type name
+changes from WellKnownCACertType to `WellKnownCACertificatesType`
+- the constant WellKnownCACertSystem becomes `WellKnownCACertificatesSystem`
+
+Leading Contributors: Candace Holman, Rob Scott
+
+### Everything else
+
+For a full list of the changes included in this release, please refer to the
+[v1.1.0 release
+notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.1.0).
+
+## Gateway API background
+
+The idea of Gateway API was initially [proposed](https://youtu.be/Ne9UJL6irXY?si=wgtC9w8PMB5ZHil2)
+at the 2019 KubeCon San Diego as the next generation
+of Ingress API. Since then, an incredible community has formed to develop what
+has likely become the
+[most collaborative API in Kubernetes history](https://www.youtube.com/watch?v=V3Vu_FWb4l4).
+Over 200 people have contributed to this API so far, and that number continues
+to grow.
+
+The maintainers
+would like to thank _everyone_ who's contributed to Gateway API, whether in the
+form of commits to the repo, discussion, ideas, or general support. We literally
+couldn't have gotten this far without the support of this dedicated and active
+community.
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+one of the 5 most recent minor versions of Kubernetes (1.25+), you'll be able to
```

Comment:
```suggestion
a version of Kubernetes  >= 1.25+, you'll be able to
```

This make sense still?
I'm thinking this will stand the test of time as new releases come out

Reply:
I think Gateway is making a supportability promise here. Not sure.

Reply:
I copy/pasted from previous Gateway API blogs.  @robscott is it worth keeping?

Reply:
Yeah, I'd keep something like this in place. Technically we should bump the version to v1.26+ in this post now that v1.30 is out. In general, I think the key thing we're trying to communicate is that Gateway API is committed to supporting the 5 most recent minor versions of Kubernetes, so I think we should make sure the wording we choose here makes that part clear.

Reply:
Since this is for humans, how about 

> As long as you're running Kubernetes 1.26 or later...

?

Reply:
Updated.